website/content/guides/1.7-plugin-upgrade/index.mdx

```diff
@@ -138,11 +140,22 @@ func main() {
 
 ```
 
-So the implementation is extremely similar, but now we use the "NewSet" function, and call Run() instead of Serve().
+The implementation is extremely similar, but now we use the "NewSet" function to create the plugin Server, and call Run() instead of Serve().
+
+With the multi-plugin binary example, it is important to not that the name of the binary must be in the form of `packer-pluign-bar`. Otherwise Packer will not register it as a multiple plugin.
```

Comment:
```suggestion
With the multi-plugin binary example, it is important to note that the name of the binary must be in the form of `packer-pluign-name`. Otherwise Packer will not register it as a multi-plugin.
```